### ISSUE TITLE
Разделить Handlers на соответствующие сервисы

### ISSUE THREAD

**Issue by Alex2021cyberhuman** [brown-aleks/CleannetCode_bot#4]
Реализация методов Handlers может быть разделена на классы.
Это следует сделать, так как методы не зависят друг от друга и их существование в одном классе походит на анти-паттерн "God Object".

**Comment by brown-aleks** [comment]
Согласен... мне этот класс, тоже не нравится. Я от него избавился вовсе. В следующем коммите предложена несколько иной конструктивный подход. Буду очень благодарен, если к нему вы тоже оставите своё мнение в комментариях.
